patches/chrome-installer-util-shell_util.cc.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/chrome/installer/util/shell_util.cc b/chrome/installer/util/shell_util.cc
+index bba833d2f3abeb8f0a6fefa59ace416db3dc2b63..ca3604b2ab8fcd9a74718e54fb1263d4ba32be26 100644
+--- a/chrome/installer/util/shell_util.cc
++++ b/chrome/installer/util/shell_util.cc
+@@ -748,6 +748,10 @@ base::string16 GetTargetForDefaultAppsSettings(const wchar_t* protocol) {
+     return base::StringPrintf(kSystemSettingsDefaultAppsFormat, L"Browser");
+   if (base::EqualsCaseInsensitiveASCII(protocol, L"mailto"))
+     return base::StringPrintf(kSystemSettingsDefaultAppsFormat, L"Email");
++  if (base::EqualsCaseInsensitiveASCII(protocol, L"ipfs"))
++    return base::StringPrintf(kSystemSettingsDefaultAppsFormat, L"IPFS");
++  if (base::EqualsCaseInsensitiveASCII(protocol, L"ipns"))
++    return base::StringPrintf(kSystemSettingsDefaultAppsFormat, L"IPNS");
```

Comment:
I would suggest putting these in a #define BRAVE_GET_TARGET_FOR_DEFAULT_APP_SETTINGS in a `chromium_src` override file and just patching in the #define name in the patch.